server/public/styles.css

```diff
@@ -0,0 +1,53 @@
+* {
+  font-family: Roboto, sans-serif;
+}
+
+.form-title {
+  margin-top: 43px;
+  font-size: 2.3rem;
+}
```

Comment:
If everything is going to be 43 px from the top, just add a padding top of 43px to the body element so you don't have to put this margin on every single page title